src/translations/en.json

```diff
@@ -190,6 +190,11 @@
       "gas": "Gas",
       "manual": "Manual"
     },
+    "climate_fan": {
```

Comment:
Please don't add the translations to the `state` section, that is reserved for states instead of attributes. Can you add a new section, `state_attributes`:

```json
"state_attributes": {
  "climate": {
    "fan": {
            "off": "Off",
            "on": "On",
            "auto": "Auto"
    }
  }
}
```

Reply:
Change made.  Used: `fan_mode` instead of `fan` to avoid confusion elsewhere.